docs/user-guide/deployment-scenarios.md

```diff
@@ -0,0 +1,51 @@
+# Deployment Scenarios
+
+The StarterKit support a wide variety of deployments, and this article will
```

Comment:
We use different wording for the starter kit in different places, maybe we should homogenize this?